include/swift/SIL/SILCloner.h

```diff
@@ -1252,15 +1297,28 @@ void
 SILCloner<ImplClass>::visitWitnessMethodInst(WitnessMethodInst *Inst) {
   auto conformance =
     getOpConformance(Inst->getLookupType(), Inst->getConformance());
+  auto lookupType = Inst->getLookupType();
+  auto newLookupType = getOpASTType(lookupType);
+  SILValue OpenedExistential;
+  if (Inst->hasOperand())
+    OpenedExistential = getOpValue(Inst->getOperand());
```

Comment:
Could you be more explicit when getting the opened archetype operand by using getNumOpenedArchetypesOperands(), getOpenedArchetypesOperands?

It looks like this assumes that if the old instruction has an opened archetype operand, that the new one will too. I'm not sure that's right. If so, then we should have some assert.